### PR TITLE
Drop 0.6

### PR DESCRIPTION
**Pull request by abelsiqueira**
Don't merge yet, waiting for #64.

### CODE REVIEW

**Comment by dpo** [comment]
Because this package is not registered, we should probably tag a release before merging this. But I'd like to merge the other PRs first.

File: src/krylov_aux.jl [hunk 1]
```diff
@@ -123,30 +123,30 @@ end
...
+function krylov_axpy!(n :: Int, s :: T, x :: Vector{T}, dx :: Int, y :: Vector{T}, dy :: Int) where T <: Number
   # assume dx = dy
   @simd for i = 1:dx:n
     @inbounds y[i] += s * x[i]
   end
   return y
 end
 
-function krylov_axpy!{T <: Number}(n :: Int, s :: T, x :: Vector{T}, dx :: Int, t :: T, y :: Vector{T}, dy :: Int)
+function krylov_axpy!(n :: Int, s :: T, x :: Vector{T}, dx :: Int, t :: T, y :: Vector{T}, dy :: Int) where T <: Number
```
Comment (Xx-fractale-xX) [inline src/krylov_aux.jl]: They add BLAS.axpby! function in julia 0.7.
```julia
krylov_axpby!{T <: BLAS.BlasReal}(n :: Int, s :: T, x :: Vector{T}, dx :: Int, t :: T, y :: Vector{T}, dy :: Int) = BLAS.axpby!(n, s, x, dx, t, y, dy)
function krylov_axpby!{T <: Number}(n :: Int, s :: T, x :: Vector{T}, dx :: Int, t :: T, y :: Vector{T}, dy :: Int)
  # assume dx = dy
  @simd for i = 1:dx:n
    @inbounds y[i] = s * x[i] + t * y[i]
  end
  return y
end
```

Comment (Xx-fractale-xX) [reply src/krylov_aux.jl]:
```julia
macro kaxpby!(n, s, x, t, y)
  return esc(:(krylov_axpby!($n, $s, $x, 1, $t, $y, 1)))
end
```
It's shorter than a `@kscal!` + `@kaxpy!`.

**Comment by dpo** [comment]
@abelsiqueira Could you rebase this?

**Comment by dpo** [comment]
Rebased and furthered in #72.

**Comment by abelsiqueira** [comment]
Thanks

**Pull request closed without merging**